src/callbacks/onResize/tickClick.js

```diff
@@ -1,27 +1,16 @@
-import drawParticipantTimeline from '../functions/drawParticipantTimeline';
+import drawIDtimeline from '../functions/drawIDtimeline';
+import updateIDfilter from '../functions/updateIDfilter';
+import enableDisableControls from '../functions/enableDisableControls';
 
 export default function tickClick() {
-    drawParticipantTimeline.call(this);
+    drawIDtimeline.call(this);
+    enableDisableControls.call(this);
+    updateIDfilter.call(this);
 
-    //Update participant filter.
+    //Highlight ID dropdown.
     this.controls.wrap
```

Comment:
Don't think I'm seeing anything in bold.

Reply:
Minor formatting issue - fix here if it's easy.

Reply:
It's only briefly bold.

Reply:
Clicking an ID or the back button brings the focus to the ID dropdown and briefly bolds its label.